winui/Kanban/getting-started.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: post
+title: Getting Started with WinUI Kanban control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Kanban (SfKanban) control, its elements, and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Getting Started with WinUI Kanban Control
+
+This section provides a quick overview of how to get started with the WinUI Kanban control (SfKanban) for WinUI and a walk-through to configure the WinUI Kanban control in a real-time scenario. Follow the steps below to add WinUI Kanban control to your project.
+
+## Creating an application with WinUI Kanban
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app).
```

Comment:
[WinUI 3 desktop app for C# and .NET 6](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app).